Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.h

```diff
@@ -298,6 +299,11 @@ namespace AzToolsFramework
         void SetEntityLocalRotation(AZ::EntityId entityId, const AZ::Vector3& localRotation);
         void SetEntityLocalRotation(AZ::EntityId entityId, const AZ::Quaternion& localRotation);
 
+        void PerformGroupDitto(AZ::EntityId entityId);
+        void PerformIndividualDitto(AZ::EntityId entityId);
+        void PerformManipulatorDitto(AZ::EntityId entityId);
+        void PerformSnapToTerrain(const ViewportInteraction::MouseInteractionEvent& mouseInteraction);
```

Comment:
[suggestion] I'm not a native English speaker, but isn't Ditto like an informal word? What does it mean to ditto a group?
What about some more specific word like `Copy` or `Mimic`. Maybe specify what is "dittoing" would be useful, for example if it's actions it could be `CopyActions` or `MimicActions`.

Reply:
Ditto isn't a slang/informal word as far as I know, it appears here - https://www.merriam-webster.com/dictionary/ditto (it was just corrupted by Pokemon last I checked 😝)

We went back and forward on this a bunch (when Chris was still here) - Copy doesn't really work as you're not duplicating anything (and Copy often goes with Paste too). We had thought of `Match` and `Mimic` but went with `Ditto` as it does accurately reflect what is happening.

The word 'ditto' is used in the docs (for example here - https://o3de.org/docs/user-guide/editor/viewport/transform-matching/) so I'd rather not start changing this as part of this change. I'll discuss with @amzn-leenguy when we get to the documentation phase and see if we do want to update the naming at all

Reply:
Thanks Tom for you detailed explanation. I didn't know it was officially called "ditto feature". I see in the documentation it's "to share an entity’s transform data from one entity to another". Maybe a comment in the code wouldn't hurt, it's possible you already have it somewhere else that is not part of this PR, in a class or something, if not, consider adding it.